nirc_ehr/resources/views/populateData.html

```diff
@@ -229,7 +250,44 @@
                 if (this.pendingInserts==0){
                     document.getElementById('msgbox').innerHTML += '<div>Populate Complete</div>';
                 }
-            }
+            },
+
+            populateLookupSets: function () {
+                this.pendingInserts++;
+
+                //records for reports:
+                var config = {
+                    schemaName: 'ehr_lookups',
+                    queryName: 'lookup_sets',
+                    moduleResource: '/data/lookup_sets.tsv',
+                    success: this.onSuccess,
+                    failure: this.onError,
+                    scope: this
+                };
+
+                var origSuccess = config.success;
+                config.success = function (results, xhr, c) {
+                    console.log('lookup set records inserted');
+
+                    LABKEY.Ajax.request({
```

Comment:
Is this required? I would have expected your changes here to make this unnecessary:

https://github.com/LabKey/ehrModules/pull/250

Reply:
I forgot about that. This has been removed